tests/tests_pytorch/helpers/runif.py

```diff
@@ -11,193 +11,4 @@
-            except (AssertionError, RuntimeError) as e:
-                # AssertionError: Torch not compiled with CUDA enabled
-                # RuntimeError: Found no NVIDIA driver on your system.
-                is_unrelated = "Found no NVIDIA driver" not in str(e) or "Torch not compiled with CUDA" not in str(e)
-                if is_unrelated:
-                    raise e
-                cond = True
-
-            conditions.append(cond)
-            reasons.append("CUDA device bf16")
-
-        if skip_windows:
-            conditions.append(sys.platform == "win32")
-            reasons.append("unimplemented on Windows")
-
-        if tpu:
-            conditions.append(not TPUAccelerator.is_available())
-            reasons.append("TPU")
-            # used in conftest.py::pytest_collection_modifyitems
-            kwargs["tpu"] = True
-
-        if ipu:
-            conditions.append(not _IPU_AVAILABLE)
-            reasons.append("IPU")
-            # used in conftest.py::pytest_collection_modifyitems
-            kwargs["ipu"] = True
-
-        if hpu:
-            conditions.append(not _HPU_AVAILABLE)
-            reasons.append("HPU")
-
-        if mps is not None:
-            if mps:
-                conditions.append(not MPSAccelerator.is_available())
-                reasons.append("MPS")
-            else:
-                conditions.append(MPSAccelerator.is_available())
-                reasons.append("not MPS")
-
-        if standalone:
-            env_flag = os.getenv("PL_RUN_STANDALONE_TESTS", "0")
-            conditions.append(env_flag != "1")
-            reasons.append("Standalone execution")
-            # used in conftest.py::pytest_collection_modifyitems
-            kwargs["standalone"] = True
-
-        if deepspeed:
-            conditions.append(not _DEEPSPEED_AVAILABLE)
-            reasons.append("Deepspeed")
-
-        if rich:
-            conditions.append(not _RICH_AVAILABLE)
-            reasons.append("Rich")
-
-        if omegaconf:
-            conditions.append(not _OMEGACONF_AVAILABLE)
-            reasons.append("omegaconf")
-
-        if psutil:
-            conditions.append(not _PSUTIL_AVAILABLE)
-            reasons.append("psutil")
-
-        if sklearn:
-            conditions.append(not _SKLEARN_AVAILABLE)
-            reasons.append("scikit-learn")
-
-        if onnx:
-            conditions.append(_TORCH_GREATER_EQUAL_2_0 and not _ONNX_AVAILABLE)
-            reasons.append("onnx")
-
-        reasons = [rs for cond, rs in zip(conditions, reasons) if cond]
-        return pytest.mark.skipif(
-            *args, condition=any(conditions), reason=f"Requires: [{' + '.join(reasons)}]", **kwargs
-        )
-
-
-@RunIf(min_torch="99")
-def test_always_skip():
-    exit(1)
-
-
-@pytest.mark.parametrize("arg1", [0.5, 1.0, 2.0])
-@RunIf(min_torch="0.0")
-def test_wrapper(arg1: float):
-    assert arg1 > 0.0
+from lightning.pytorch.utilities.testing import _RunIf as RunIf  # noqa: F401
```

Comment:
@Borda I did this so that we don't create a thousand changes in the tests. This also won't invalidate other PRs that use the old import